.eslintrc.json

```diff
@@ -32,15 +32,19 @@
     "Generator": false
   },
   "rules": {
-    "quotes": ["error", "double", { "avoidEscape": true }],
-    "max-len": ["off", 100],
+    "array-callback-return": "warn",
```

Comment:
I sorted the rules alphabetically, which is why there are so many changes. I've also added a couple of rules here. Easiest way to check (if you want to) is probably to sort the rules from the old file alphabetically and then do a diff :/